rjit.c

```diff
@@ -0,0 +1,35 @@
+#include "internal/symbol.h"
+#include "ruby.h"
+#include <stdlib.h>
+#include <stdio.h>
+
+#if USE_YJIT
+typedef void * (*intern_cstr_func)(const char *);
+
+size_t rb_rjit_address_of(void * name, intern_cstr_func func);
+
+static VALUE
+address_of(VALUE m, VALUE name)
+{
+    size_t addr = rb_rjit_address_of((void *)name, (intern_cstr_func)rb_sym_intern_ascii_cstr);
+
+    if (addr) {
+        return SIZET2NUM(addr);
+    }
+    else {
+        return Qfalse;
```

Comment:
Nitpick but would `Qnil` be more in keeping with Ruby conventions?

Reply:
> Nitpick but would `Qnil` be more in keeping with Ruby conventions?

I'm not sure. I have no strong opinion, so I'll change to `Qnil`.